sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/ClientCommon.cs

```diff
@@ -125,5 +128,15 @@ public static Guid GetFeedbackId(string locationHeader)
                 throw new ArgumentException(UnexpectedHeaderFormat);
             }
         }
+
+        internal static HookInfoPatch GetPatchModel(AlertingHook hook)
```

Comment:
Any reason why this is in `ClientCommon` and not in `AlertingHook` ?

Reply:
Good question - I think it evolved after some refactoring, but I agree it should move.